newt/syscfg/syscfg.go

```diff
@@ -1009,6 +1009,15 @@ func (cfg *Cfg) detectAmbiguities() {
 	}
 }
 
+// Detects and records errors in the build's syscfg.  This should only be
+// called after APIs are resolved to avoid false positives.
+func (cfg *Cfg) DetectErrors(flashMap flash.FlashMap) {
```

Comment:
Wouldn't it be better to receive `flashMap` as pointer?

Reply:
Yeah, that's a good question.  I never really know whether to pass by value or pointer in Go.  These days I tend to pass by value unless the function is intended to modify the data since there is no `const`.